install.sh

```diff
@@ -30,56 +34,50 @@ else
   USER_SPACE_ARCH="${ARCH}"
 fi
 
-# BOOTSTRAP_PACKAGES cannot depend on crew_profile_base for their core operations (completion scripts are fine)
-BOOTSTRAP_PACKAGES="crew_mvdir pixz ca_certificates ruby openssl"
-[ -x /usr/bin/zstd ] || BOOTSTRAP_PACKAGES="zstd ${BOOTSTRAP_PACKAGES}" # use system zstd if available
-[ -x "${CREW_PREFIX}"/bin/xz ] && rm "${CREW_PREFIX}"/bin/xz # remove local xz if installed
-
-RED='\e[1;91m';    # Use Light Red for errors.
-YELLOW='\e[1;33m'; # Use Yellow for informational messages.
-GREEN='\e[1;32m';  # Use Green for success messages.
-BLUE='\e[1;34m';   # Use Blue for intrafunction messages.
-GRAY='\e[0;37m';   # Use Gray for program output.
-MAGENTA='\e[1;35m';
+if [[ "$ARCH" == "x86_64" ]]; then
+  LIB_SUFFIX='64'
+fi
+
+
 RESET='\e[0m'
 
 # Simplify colors and print errors to stderr (2).
-echo_error() { echo -e "${RED}${*}${RESET}" >&2; }
-echo_info() { echo -e "${YELLOW}${*}${RESET}" >&1; }
-echo_success() { echo -e "${GREEN}${*}${RESET}" >&1; }
-echo_intra() { echo -e "${BLUE}${*}${RESET}" >&1; }
-echo_out() { echo -e "${GRAY}${*}${RESET}" >&1; }
-echo_other() { echo -e "${MAGENTA}${*}${RESET}" >&2; }
+echo_error() { echo -e "\e[1;91m${*}${RESET}" >&2; } # Use Light Red for errors.
+echo_info() { echo -e "\e[1;33m${*}${RESET}" >&1; } # Use Yellow for informational messages.
+echo_success() { echo -e "\e[1;32m${*}${RESET}" >&1; } # Use Green for success messages.
+echo_intra() { echo -e "\e[1;34m${*}${RESET}" >&1; } # Use Blue for intrafunction messages.
+echo_out() { echo -e "\e[0;37m${*}${RESET}" >&1; } # Use Gray for program output.
 
 # Skip all checks if running on a docker container.
 [[ -f "/.dockerenv" ]] && CREW_FORCE_INSTALL=1
 
 # Reject crostini.
 if [[ -d /opt/google/cros-containers && "${CREW_FORCE_INSTALL}" != '1' ]]; then
   echo_error "Crostini containers are not supported by Chromebrew :/"
-  echo_info "Run 'curl -Ls git.io/vddgY | CREW_FORCE_INSTALL=1 bash' to perform install anyway"
+  echo_info "Run 'CREW_FORCE_INSTALL=1 exec bash --init-file <(curl -Ls git.io/vddgY)' to perform install anyway"
   exit 1
 fi
 
-# Disallow non-stable channels Chrome OS.
+# Reject non-stable Chrome OS channels.
 if [ -f /etc/lsb-release ]; then
   if [[ ! "$(< /etc/lsb-release)" =~ CHROMEOS_RELEASE_TRACK=stable-channel$'\n' && "${CREW_FORCE_INSTALL}" != '1' ]]; then
     echo_error "The beta, dev, and canary channel are unsupported by Chromebrew"
-    echo_info "Run 'curl -Ls git.io/vddgY | CREW_FORCE_INSTALL=1 bash' to perform install anyway"
+    echo_info "Run 'CREW_FORCE_INSTALL=1 exec bash --init-file <(curl -Ls git.io/vddgY)' to perform install anyway"
```

Comment:
How is this better?

Reply:
We can take user input this way, which we need for choosing whether to clear CREW_PREFIX or not.